src/value/mod.rs

```diff
@@ -1,21 +1,26 @@
 use std::borrow::Cow;
 use std::cmp::Ordering;
-use std::collections::HashMap;
-use std::collections::{BTreeMap, BTreeSet};
+use std::collections::BTreeMap;
+use std::collections::BTreeSet;
+use hashbrown::HashMap;
 use std::fmt;
 use std::fmt::Formatter;
 use std::sync::Arc;
 
 #[cfg(feature = "preserve_order")]
 use indexmap::IndexMap;
-use serde::ser::{Serialize, SerializeMap, SerializeSeq, Serializer};
+use serde::ser::Serialize;
+use serde::ser::SerializeMap;
+use serde::ser::SerializeSeq;
+use serde::ser::Serializer;
```

Comment:
All those changes seem unrelated. It's a bit annoying to have one line per import, can you revert those?

Reply:
I did it to be able to use a sed command to change the HashMap impl. I'll fix up all these mistakes.

I made the PR late at night and forgot to clean things up 😅